samples/bluetooth/peripheral_ht/src/hts.c

```diff
@@ -53,26 +56,65 @@ BT_GATT_SERVICE_DEFINE(hts_svc,
 
 void hts_init(void)
 {
+	temp_dev = device_get_binding("TEMP_0");
+
+	if (!temp_dev) {
+		printk("error: no temp device\n");
+		return;
+	}
+
+	printk("temp device is %p, name is %s\n", temp_dev,
+	       temp_dev->config->name);
 }
 
 void hts_indicate(void)
 {
 	/* Temperature measurements simulation */
+	struct sensor_value temp_value;
+
 	if (simulate_htm) {
+		static u8_t htm[5];
+		double temperature;
```

Comment:
temperature needs to be static variable for simulated values. And assign to 20U.

Reply:
assigned 20U value for temperature and its working fine